--- origsrc/valadoc/src/doclets/devhelp/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/devhelp/Makefile.am	2010-05-12 22:02:43.735002300 -0500
@@ -37,7 +37,7 @@ libdoclet_la_SOURCES =      \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/gtkdoc/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/gtkdoc/Makefile.am	2010-05-12 22:02:52.582508300 -0500
@@ -42,7 +42,7 @@ libdoclet_la_SOURCES =  \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc  --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc  --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/htm/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/htm/Makefile.am	2010-05-12 22:02:58.278834100 -0500
@@ -37,7 +37,7 @@ libdoclet_la_SOURCES =  \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc  --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc  --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/valadoc.org/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/valadoc.org/Makefile.am	2010-05-12 22:12:08.298293400 -0500
@@ -11,7 +11,7 @@ BUILT_SOURCES = libdoclet.vala.stamp
 
 
 libdoclet.vala.stamp: $(libdoclet_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/libvaladoc/ --vapidir $(top_srcdir)/src/vapi/ --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . --save-temps $^
+	$(VALAC) -C --vapidir $(top_builddir)/src/libvaladoc/ --vapidir $(top_srcdir)/src/vapi/ --pkg vala-1.0 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . --save-temps $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/xml/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/xml/Makefile.am	2010-05-12 22:03:13.607710900 -0500
@@ -12,7 +12,7 @@ BUILT_SOURCES = libdoclet.vala.stamp
 
 
 libdoclet.vala.stamp: $(libdoclet_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi/ --vapidir $(top_srcdir)/src/libvaladoc --pkg valadoc-1.0 --pkg vala-1.0 --pkg gee-1.0 --basedir . --save-temps $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi/ --vapidir $(top_builddir)/src/libvaladoc --pkg valadoc-1.0 --pkg vala-1.0 --pkg gee-1.0 --basedir . --save-temps $^
 	touch $@
 
 
--- origsrc/valadoc/src/libvaladoc/Makefile.am	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/libvaladoc/Makefile.am	2010-05-12 22:09:09.764081900 -0500
@@ -14,7 +14,7 @@ AM_CFLAGS = \
 	$(NULL)
 
 
-BUILT_SOURCES = libvaladoc.vala.stamp
+BUILT_SOURCES = libvaladoc.vala.stamp $(abs_builddir)/valadoc-1.0.deps
 
 
 lib_LTLIBRARIES = \
@@ -181,3 +181,5 @@ vapi_DATA =     \
 	$(NULL)
 
 
+$(abs_builddir)/valadoc-1.0.deps: $(abs_srcdir)/valadoc-1.0.deps
+	cp -f $< $@
