--- origsrc/valadoc/src/libvaladoc/moduleloader.vala	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/libvaladoc/moduleloader.vala	2010-05-12 21:53:10.946240600 -0500
@@ -49,7 +49,7 @@ public class Valadoc.ModuleLoader : Type
 	public bool load_doclet (string path) {
 		void* function;
 
-		docletmodule = Module.open (Module.build_path (path, "libdoclet"), ModuleFlags.BIND_LAZY | ModuleFlags.BIND_LOCAL);
+		docletmodule = Module.open (Module.build_path (path, "doclet"), ModuleFlags.BIND_LAZY | ModuleFlags.BIND_LOCAL);
 		if (docletmodule == null) {
 			return false;
 		}
