--- origsrc/valadoc/configure.in	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/configure.in	2009-11-10 01:32:06.475948200 -0600
@@ -66,7 +66,6 @@ AC_CONFIG_FILES([Makefile
                  src/doclets/htm/Makefile
                  src/doclets/valadoc.org/Makefile
                  src/doclets/devhelp/Makefile
-                 src/doclets/xml/Makefile
                  src/valadoc/Makefile])
 
 AC_OUTPUT
--- origsrc/valadoc/src/doclets/devhelp/Makefile.am	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/src/doclets/devhelp/Makefile.am	2009-11-10 01:54:47.025391600 -0600
@@ -17,6 +17,9 @@ BUILT_SOURCES = libdoclet.vala.stamp
 docletdir = $(libdir)/valadoc/plugins/devhelp
 
 
+libdoclet_la_LDFLAGS = -module -avoid-version -no-undefined
+
+
 doclet_LTLIBRARIES =  \
 	libdoclet.la \
 	$(NULL)
@@ -34,7 +37,7 @@ libdoclet_la_SOURCES =      \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir ../../vapi --vapidir ../../libvaladoc --pkg valadoc-1.0 --pkg gee-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir ../../libvaladoc --pkg valadoc-1.0 --pkg vala-1.0 --pkg gee-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/htm/Makefile.am	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/src/doclets/htm/Makefile.am	2009-11-10 01:54:55.225402900 -0600
@@ -17,7 +17,7 @@ BUILT_SOURCES = libdoclet.vala.stamp
 docletdir = $(libdir)/valadoc/plugins/html
 
 
-libdoclet_la_LDFLAGS = -module -avoid-version
+libdoclet_la_LDFLAGS = -module -avoid-version -no-undefined
 
 
 doclet_LTLIBRARIES =  \
@@ -37,7 +37,7 @@ libdoclet_la_SOURCES =  \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir ../../vapi --vapidir ../../libvaladoc --pkg valadoc-1.0 --pkg gee-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir ../../libvaladoc --pkg valadoc-1.0 --pkg vala-1.0 --pkg gee-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/valadoc.org/Makefile.am	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/src/doclets/valadoc.org/Makefile.am	2009-11-10 01:56:54.176567100 -0600
@@ -11,7 +11,7 @@ BUILT_SOURCES = libdoclet.vala.stamp
 
 
 libdoclet.vala.stamp: $(libdoclet_VALASOURCES)
-	$(VALAC) -C --vapidir ../../libvaladoc/ --vapidir ../../vapi/ --pkg valadoc-1.0 --pkg gee-1.0 --basedir . --save-temps $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi/ --vapidir ../../libvaladoc --pkg valadoc-1.0 --pkg vala-1.0 --pkg gee-1.0 --basedir . --save-temps $^
 	touch $@
 
 
@@ -38,7 +38,7 @@ AM_CFLAGS =  -g \
 	$(NULL)
 
 
-libdoclet_la_LDFLAGS = -module -avoid-version \
+libdoclet_la_LDFLAGS = -module -avoid-version -no-undefined \
 	$(NULL)
 
 
--- origsrc/valadoc/src/libvaladoc/Makefile.am	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/src/libvaladoc/Makefile.am	2009-11-10 01:36:17.208296500 -0600
@@ -14,12 +14,10 @@ AM_CFLAGS = \
 BUILT_SOURCES = libvaladoc.vala.stamp
 
 
-valadoc_LTLIBRARIES = \
+lib_LTLIBRARIES = \
 	libvaladoc.la     \
 	$(NULL)
 
-valadocdir = $(libdir)/valadoc
-
 
 libvaladoc_la_VALASOURCES = \
 	doclet.vala \
@@ -136,10 +134,12 @@ libvaladocincludedir = $(includedir)/
 
 
 libvaladoc.vala.stamp: $(libvaladoc_la_VALASOURCES)
-	$(VALAC) $(VALAFLAGS) -C -H valadoc-1.0.h --pkg gee-1.0 --pkg vala-1.0 --pkg gmodule-2.0 --vapidir ../vapi --pkg libgvc --library valadoc-1.0 --basedir $(top_srcdir)/src/libvaladoc/ --save-temps $^
+	$(VALAC) $(VALAFLAGS) -C -H valadoc-1.0.h --pkg gee-1.0 --pkg vala-1.0 --pkg gmodule-2.0 --vapidir $(top_srcdir)/src/vapi --pkg libgvc --library valadoc-1.0 --basedir $(top_srcdir)/src/libvaladoc/ --save-temps $^
 	touch $@
 
 
+libvaladoc_la_LDFLAGS = -no-undefined
+
 libvaladoc_la_LIBADD = \
 	$(LIBGVC_LIBS)     \
 	$(LIBGEE_LIBS)     \
--- origsrc/valadoc/src/valadoc/Makefile.am	2009-11-10 01:05:36.000000000 -0600
+++ src/valadoc/src/valadoc/Makefile.am	2009-11-10 01:41:26.781285900 -0600
@@ -41,7 +41,7 @@ valadoc_LDADD =                 \
 
 
 valadoc.vala.stamp: $(valadoc_VALASOURCES)
-	$(VALAC) -C --pkg gee-1.0 --pkg gmodule-2.0 --vapidir ../vapi/ --pkg config --vapidir ../libvaladoc/ --pkg valadoc-1.0 $^
+	$(VALAC) -C --pkg vala-1.0 --pkg gee-1.0 --pkg gmodule-2.0 --vapidir $(top_srcdir)/src/vapi/ --pkg config --vapidir ../libvaladoc/ --pkg valadoc-1.0 $^
 	touch $@
 
 
