--- origsrc/valadoc/src/doclets/gtkdoc/doclet.vala	2013-01-13 20:43:57.000000000 -0600
+++ src/valadoc/src/doclets/gtkdoc/doclet.vala	2013-01-13 20:53:55.949353100 -0600
@@ -281,21 +281,21 @@ public class Gtkdoc.Director : Valadoc.D
 		pc_libs += "--libs";
 
 		try {
-			string stderr;
+			string cerr;
 			int status;
 
 			string cflags;
-			Process.spawn_sync (null, pc_cflags, null, SpawnFlags.SEARCH_PATH, null, out cflags, out stderr, out status);
+			Process.spawn_sync (null, pc_cflags, null, SpawnFlags.SEARCH_PATH, null, out cflags, out cerr, out status);
 			if (status != 0) {
-				reporter.simple_error ("GtkDoc: error: pkg-config cflags error: %s", stderr);
+				reporter.simple_error ("GtkDoc: error: pkg-config cflags error: %s", cerr);
 				return false;
 			}
 			cflags = cflags.strip ();
 
 			string libs;
-			Process.spawn_sync (null, pc_libs, null, SpawnFlags.SEARCH_PATH, null, out libs, out stderr, out status);
+			Process.spawn_sync (null, pc_libs, null, SpawnFlags.SEARCH_PATH, null, out libs, out cerr, out status);
 			if (status != 0) {
-				reporter.simple_error ("GtkDoc: error: pkg-config libs error: %s", stderr);
+				reporter.simple_error ("GtkDoc: error: pkg-config libs error: %s", cerr);
 				return false;
 			}
 
