--- origsrc/valadoc/src/doclets/devhelp/Makefile.am	2011-11-16 02:36:16.000000000 -0600
+++ src/valadoc/src/doclets/devhelp/Makefile.am	2011-11-16 02:42:04.358462200 -0600
@@ -36,7 +36,7 @@ libdoclet_la_SOURCES =      \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/gtkdoc/Makefile.am	2011-11-16 02:36:16.000000000 -0600
+++ src/valadoc/src/doclets/gtkdoc/Makefile.am	2011-11-16 02:42:36.658309600 -0600
@@ -41,7 +41,7 @@ libdoclet_la_SOURCES =  \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/doclets/htm/Makefile.am	2011-11-16 02:36:16.000000000 -0600
+++ src/valadoc/src/doclets/htm/Makefile.am	2011-11-16 02:42:46.231857200 -0600
@@ -36,7 +36,7 @@ libdoclet_la_SOURCES =  \
 
 
 libdoclet.vala.stamp: $(libdoclet_la_VALASOURCES)
-	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_srcdir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(top_builddir)/src/libvaladoc --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/driver/0.14.x/Makefile.am	2011-11-16 02:36:16.000000000 -0600
+++ src/valadoc/src/driver/0.14.x/Makefile.am	2011-11-16 02:53:50.776867000 -0600
@@ -43,7 +43,7 @@ libdriver_la_SOURCES =      \
 
 
 libdriver.vala.stamp: $(libdriver_la_VALASOURCES)
-	$(VALAC) $(VALA_FLAGS) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(VERSIONED_VAPI_DIR) --vapidir $(top_srcdir)/src/libvaladoc --pkg libvala-0.14 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
+	$(VALAC) $(VALA_FLAGS) -C --vapidir $(top_srcdir)/src/vapi --vapidir $(VERSIONED_VAPI_DIR) --vapidir $(top_builddir)/src/libvaladoc --pkg libvala-0.14 --pkg gee-1.0 --pkg valadoc-1.0 --basedir . $^
 	touch $@
 
 
--- origsrc/valadoc/src/libvaladoc/Makefile.am	2011-11-16 02:36:16.000000000 -0600
+++ src/valadoc/src/libvaladoc/Makefile.am	2011-11-16 02:43:59.201030800 -0600
@@ -11,7 +11,7 @@ AM_CFLAGS = \
 	$(NULL)
 
 
-BUILT_SOURCES = libvaladoc.vala.stamp
+BUILT_SOURCES = libvaladoc.vala.stamp $(abs_builddir)/valadoc-1.0.deps
 
 
 lib_LTLIBRARIES = \
